proto/manifest.proto

```diff
@@ -12,22 +12,26 @@ message Resource {
     // Path specifies the path from the bundle root. If more than one
     // path is present, the entry may represent a hardlink, rather than using
     // a link target. The path format is operating system specific.
+    // Path MAY be an absolute path, but it MUST be interpreted with the bundle root.
     repeated string path = 1;
 
     // NOTE(stevvooe): Need to define clear precedence for user/group/uid/gid precedence.
 
     // Uid specifies the user id for the resource. A string type is used for
     // compatibility across different OS.
+    // On Unix, uid MUST be a positive decimal integer or an empty string.
     string uid = 2;
```

Comment:
> please elaborate your plan for windows?

You'll have to ask the microsoft folks, but they made the indication that we are going to use separate fields for that purpose. The only reason these are strings is due to the fact Windows has string user ids.

> not uint64?

Don't use integer types to enforce signedness.